README.md

```diff
@@ -621,25 +635,25 @@ scala> val exchangeRates = List(USD / CAD(1.05), USD / MXN(12.50), USD / JPY(100
 exchangeRates: List[squants.market.CurrencyExchangeRate] = List(USD/CAD 1.05, USD/MXN 12.5, USD/JPY 100.0)
 
 scala> implicit val moneyContext = defaultMoneyContext withExchangeRates exchangeRates
-moneyContext: squants.market.MoneyContext = MoneyContext(squants.market.USD$@6678b23a,Set(squants.market.SEK$@67d53899, squants.market.DKK$@4d2a2ac3, squants.market.MYR$@2c71456a, squants.market.CLP$@14b462f8, squants.market.RUB$@41296fc3, squants.market.NZD$@77a98b1c, squants.market.INR$@5a2d7719, squants.market.BRL$@2c823f4f, squants.market.MXN$@1d557c66, squants.market.GBP$@59175872, squants.market.ARS$@31085325, squants.market.CZK$@63ff9756, squants.market.EUR$@7abe223c, squants.market.CHF$@7caa5a30, squants.market.XAU$@b0e83a6, squants.market.HKD$@464d2121, squants.market.JPY$@510901a2, squants.market.AUD$@19fe310c, squants.market.XAG$@30441fd9, squants.market.NOK$@356c5557, squants.market.CNY$@29714f66, squants.market.BTC$@1d3b273, squants.market.CAD$@f48a187, squants.market.USD$@...
+moneyContext: squants.market.MoneyContext = MoneyContext(squants.market.USD$@dfad661,Set(squants.market.GBP$@69f4ad73, squants.market.BRL$@7d0734cf, squants.market.CNY$@bce5be3, squants.market.USD$@dfad661, squants.market.HKD$@5959188e, squants.market.DKK$@259bd19, squants.market.RUB$@11b0980e, squants.market.BTC$@79056cc3, squants.market.ARS$@725ae543, squants.market.SEK$@6562b9fe, squants.market.XAG$@431dce40, squants.market.CZK$@360576, squants.market.CAD$@2695bcd0, squants.market.MXN$@3cfd1c9a, squants.market.KRW$@2df1ecc2, squants.market.NZD$@45fd55e7, squants.market.CHF$@3c035bc, squants.market.CLP$@4fabbf95, squants.market.NOK$@67c83220, squants.market.JPY$@61224aa0, squants.market.XAU$@6c60e9a9, squants.market.MYR$@14cc0d53, squants.market.EUR$@101cfdd0, squants.market.INR$@1627...
```

Comment:
I added a toString method to Currency that will output just the code.  That should stabilize the README output.